lib/src/builder/execute.rs

```diff
@@ -174,9 +175,16 @@ impl TxExecStrategy for TkoTxExecStrategy {
         // track the actual tx number to use in the tx/receipt trees as the key
         let mut actual_tx_no = 0usize;
         let num_transactions = transactions.len();
+        println!("total Txs: {}", num_transactions);
         for (tx_no, tx) in take(&mut transactions).into_iter().enumerate() {
             if !is_optimistic {
+                #[cfg(not(feature = "sp1-cycle-tracker"))]
```

Comment:
does the cycle printing ever get to the screen? If not, if we only print the cycle stuff when
```
[cfg(all(
    not(all(target_os = "zkvm", target_vendor = "succinct")),
    feature = "sp1-cycle-tracker"
))]
```

so it only prints it in when running inside the zkVM?